hal/src/main/native/athena/REVPDH.cpp

```diff
@@ -497,9 +498,15 @@ void HAL_GetREVPDHVersion(HAL_REVPDHHandle handle,
     return;
   }
 
-  HAL_ReadCANPacketTimeout(hpdh->hcan, PDH_VERSION_FRAME_API, packedData,
-                           &length, &timestamp, kDefaultControlPeriod * 2,
-                           status);
+  uint32_t timeoutMs = 100;
```

Comment:
Don't do this. We don't want this to be a blocking call. What we had was fine, especially since it's not a 100% critical function

Reply:
What we had before didn't work, because we were trying to receive a packet that hadn't been sent yet. With RTR packets, you have to wait a little bit of time for a response.

I looked to see how other code in WPILib handles RTR packets, but unfortunately there are no other callers of `HAL_WriteCANRTRFrame` for me to reference.

Reply:
Oh, I didn't realize it was an rtr frame. I'm a bit more on with this, but I'd probably cache the value if we ever get a valid value. Since the returned value will never change over the life of the program.

Reply:
Sure, I can do that.

On a semi-unrelated note, why are the `PowerDistribution.getVersion()` and `PneumaticHub.getVersion()` methods package-private in Java, when their C++ counterparts are public?

Reply:
That's a bug.. all of those functions should be public. Including the fault ones.

Reply:
Should I just go ahead and fix that in this PR?

Reply:
I'll do it in a separate pr, as I likely messed up similarly in the ctre API too